### ISSUE TITLE
QR Code Scanner

### ISSUE THREAD

**Issue by rishav-karanjit** [prathimacode-hub/Awesome_Python_Scripts#1087]
Title-: QR Code Scanner
About:- My python script will be able to generate and scan QR code from an image. User will have 2 options: one to generate and another is to scan QR code from image.
Name: Rishav Karanjit
Label: Basic Python Scripts
Assignee: ""

Please assign this to me @prathimacode-hub

**Comment by vivekthedev** [comment]
@rishav-karanjit Both are present in the GUI Scripts 
[Scanner](https://github.com/prathimacode-hub/Awesome_Python_Scripts/tree/main/GUIScripts/QR%20Code%20Scanner)
[Generator](https://github.com/prathimacode-hub/Awesome_Python_Scripts/tree/main/GUIScripts/QR%20Code%20Generator)

**Comment by prathimacode-hub** [comment]
Yes @rishav-karanjit. Glad to see your initiation to contribute. It's already done. You can either enhance the existing projects or come up with new projects that doesn't exist in the repository. Before choosing a project, make sure it isn't in the repo to avoid repetitions and overcome double work. You can directly this issue message to choice you take and get it assigned. No worries.
